src/guide/single-file-component.md

```diff
@@ -1,5 +1,7 @@
 # Single File Components
 
+<VideoLesson href="https://vueschool.io/courses/vue-3-single-file-components?friend=vuejs" title="Free Vue.js Single File Components Course">Learn about single file components with a free video course on Vue School</VideoLesson>
```

Comment:
It seems that there's only one free video in this course. Should the link not point directly to that video?